pkg/cluster/createdat_test.go

```diff
@@ -0,0 +1,113 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestPopulateCreatedAt(t *testing.T) {
+	ctx := context.Background()
+	resourceID := "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resourceGroup/providers/microsoft.redhatopenshift/openshiftclusters/resourceName"
+	mockCreationTimestamp := time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
+
+	for _, tt := range []struct {
+		name     string
+		ns       *corev1.Namespace
+		wantTime time.Time
+		wantErr  string
+	}{
+		{
+			name: "default namespace exists with creation timestamp",
+			ns: &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "default",
+					CreationTimestamp: metav1.NewTime(mockCreationTimestamp),
+				},
+			},
+			wantTime: mockCreationTimestamp,
+		},
+		{
+			name: "default namespace exists without creation timestamp",
+			ns: &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "default",
+				},
+			},
+		},
+		{
+			name:    "default namespace doesn't exist",
+			wantErr: "namespaces \"default\" not found",
+			ns:      &corev1.Namespace{},
+		},
```

Comment:
Add 
`Document already has time populates`
`Document has not time populated` or something under these lines to confirm we do not override this field